roles/edpm_logrotate_crond/tasks/run.yml

```diff
@@ -27,3 +27,20 @@
       logrotate_crond: "{{ lookup('template', 'logrotate_crond.yaml.j2') | from_yaml }}"
     edpm_container_standalone_kolla_config_files:
       logrotate_crond: "{{ lookup('file', 'files/logrotate_crond.yaml') | from_yaml }}"
+  when: not ansible_local.bootc
+
+- name: Configure logrotate for bootc
+  when: ansible_local.bootc
+  become: true
+  block:
+  - name: Render kolla config file
+    ansible.builtin.copy:
+      src: files/logrotate_crond.yaml
+      dest: /var/lib/kolla/config_files/logrotate_crond.json
+      mode: 0644
+
+  - name: Manage logrotate crond containers
+    ansible.builtin.systemd_service:
+      name: edpm-compute@logrotate_crond
+      enabled: true
+      state: started
```

Comment:
I think we need to separate out this initial bootc support from the logically bound containers PR, https://github.com/openstack-k8s-operators/edpm-image-builder/pull/39

This PR has a strong dep on the logically bound PR, and that complicates things. Let's just get a base bootc working with how we manage containers presently. We can move to logically bound and all the quadlet/systemd stuff as a next step.

Reply:
Yeah, ok. Let's decouple them. I'll just submit a new PR to `edpm-image-builder` to change the Quadlet files over to using `.container` instead of `.kube`. Then a new one here to work with those instead of the `.kube` files.